features/standard/SearchContent.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Searching for a Content
```

Comment:
Nitpick: I would changes this to "Searching for Content" or "Searching for a Content item"

Reply:
Done, thx